exp/addons/api/v1beta1/clusterresourcesetbinding_types.go

```diff
@@ -100,6 +100,14 @@ func (c *ClusterResourceSetBinding) DeleteBinding(clusterResourceSet *ClusterRes
 			break
 		}
 	}
+	// Remove the ClusterResourceSet from the ClusterResourceSetBinding OwnerReferences.
+	for i, ownerRef := range c.OwnerReferences {
+		if ownerRef.Kind == clusterResourceSet.GroupVersionKind().Kind && ownerRef.Name == clusterResourceSet.Name {
+			copy(c.OwnerReferences[i:], c.OwnerReferences[i+1:])
+			c.OwnerReferences = c.OwnerReferences[:len(c.OwnerReferences)-1]
```

Comment:
Can we use util.RemoveOwnerRef here?

Reply:
Thanks for pointing to that  - I've added it now.